spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/SparkReadConf.java

```diff
@@ -243,4 +243,15 @@ public boolean preserveDataGrouping() {
         .defaultValue(SparkSQLProperties.PRESERVE_DATA_GROUPING_DEFAULT)
         .parse();
   }
+
+  public boolean aggregatePushDown() {
+    boolean enable =
+        confParser
+            .booleanConf()
+            .option(SparkReadOptions.AGGREGATE_PUSH_DOWN_ENABLED)
+            .sessionConf(SparkSQLProperties.AGGREGATE_PUSH_DOWN_ENABLED)
+            .defaultValue(SparkSQLProperties.AGGREGATE_PUSH_DOWN_ENABLED_DEFAULT)
+            .parse();
+    return enable;
```

Comment:
Do we need this temp var? Why not return directly like in other methods of this class?

```
return confParser
    .booleanConf()
    .option(SparkReadOptions.AGGREGATE_PUSH_DOWN_ENABLED)
    .sessionConf(SparkSQLProperties.AGGREGATE_PUSH_DOWN_ENABLED)
    .defaultValue(SparkSQLProperties.AGGREGATE_PUSH_DOWN_ENABLED_DEFAULT)
    .parse();
```

Reply:
Fixed. Thanks